pkg/output/term/env.go

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2024 The Knative Authors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package term
+
+import (
+	"os"
+	"strings"
+)
+
+func forceColor() bool {
+	fc := os.Getenv("FORCE_COLOR")
```

Comment:
The [`NO_COLOR`](https://no-color.org/) is a quasi standard, that's respected by numerous libraries, across various languages.

The `FORCE_COLOR` is less popular, but still, some libraries use it. The idea is to force the color output on CI, in case it supports the ANSI codes. That's why I think, we should follow those names.

Reply:
Maybe we should just reference the https://no-color.org/ and https://force-color.org/ sites in the code for posterity.

WDYT?

Reply:
Yep, on second though I realized that's more unix-driven usual way to say color `yes:no`. I'd still wrap it in our common fuction to reuse everywhere, i.e. not to spell variable name on multiple places. Otherwise OK to with the naming.

Reply:
Cool. I'll move those to a single function similar to the pseudo code from https://force-color.org/

Reply:
Fixed in 191bf3a6157fd8a3338c04de3a1632edb6f51dac